pkg/builtin/unary/date.go

```diff
@@ -0,0 +1,65 @@
+package unary
```

Comment:
could you please add the license header...

Reply:
Done